Adyen/UI/Form/Items/Text/FormTextItemStyle.swift

```diff
@@ -45,7 +45,7 @@ public struct FormTextItemStyle: FormValueItemStyle {
     
     /// The color for separator element.
     /// If value is nil, the default color would be used.
-    public var separatorColor: UIColor?
+    public var separatorColor: UIColor? = UIColor.Adyen.componentSeparator
```

Comment:
can be non-optional, right?

Reply:
it needs to be optional, because for Switch item there is no separator.